sulo/README

```diff
@@ -0,0 +1,7 @@
+# SULO Ontology
+[https://w3id.org/sulo/](https://w3id.org/sulo/) -> [https://aidava-dev.github.io/sulo/index.html](https://aidava-dev.github.io/sulo/index.html)
+
+The Simplified Upper Level Ontology
+
+## Contact
+* [Michel Dumontier - michel.dumontier@gmail.com](mailto:michel.dumontier@gmail.com)
```

Comment:
Can you add your github id here too?  That's the main thing we look at when checking changes.

Reply:
```suggestion
* [Michel Dumontier - michel.dumontier@gmail.com](mailto:michel.dumontier@gmail.com), GitHub: @micheldumontier
```